src/fortran_routing/diffusive_v02frwk/diffusive.f90

```diff
@@ -1185,7 +1162,7 @@ subroutine mesh_diffusive_forward(dtini_given, t0, t, tfin, saveInterval,j)
       
   end subroutine mesh_diffusive_forward
   
-  subroutine mesh_diffusive_backward(dtini_given, t0, t, tfin, saveInterval,j, leftBank, rightBank)
+  subroutine mesh_diffusive_backward(dtini_given, t0, t, tfin, saveInterval, j) ! leftBank, rightBank)
```

Comment:
I see we are dropping the leftBank and rightBank arguments. Are these no longer necessary arguments, even if we are using the RouteLink trapezoidal geometry data (no natural cross sections)?